app/controllers/content_items_controller.rb

```diff
@@ -67,7 +54,28 @@ def content_store
     @content_store ||= GdsApi::ContentStore.new(Plek.current.find("content-store"))
   end
 
-private
+  def configure_header_search
+    if @content_item.present? && @content_item.is_homepage?
+      remove_header_search
+    else
+      scope_header_search_to_service_manual
+    end
+  end
+
+  def scope_header_search_to_service_manual
+    # Slimmer is middleware which wraps the service manual in the GOV.UK header
+    # and footer. We set a request header so that Slimmer adds a hidden field to
+    # the header search to scope the search results to just the service manual.
+    set_slimmer_headers(
+      search_parameters: {
+        "filter_manual" => "/service-manual"
+      }.to_json,
+    )
+  end
+
+  def remove_header_search
+    set_slimmer_headers(remove_search: true)
+  end
```

Comment:
We could push these headers (both the search_parameters and remove_search) into the presenters. Maybe a `#slimmer_headers` method on the presenter classes? The benefits would be that we don't have to add the `#is_homepage?` query method because it knows what itself is, and it would remove the conditional out of the controller which is always nice.